src/components/app/QuickActions.js

```diff
@@ -74,24 +69,18 @@ export class QuickActions extends Component {
     }
   };
 
-  /**
-   * @method componentWillUnmount
-   * @summary ToDo: Describe the method
-   * @todo Write the documentation
-   */
   componentWillUnmount = () => {
     this.mounted = false;
   };
 
-  /**
-   * @method UNSAFE_componentWillReceiveProps
-   * @summary ToDo: Describe the method
-   * @param {*} nextProps
-   * @todo Write the documentation
-   */
   UNSAFE_componentWillReceiveProps = nextProps => {
     const { selected, viewId, windowType } = this.props;
 
+    if (!nextProps.viewId) {
+      console.log('MOUNT: ', viewId, nextProps.viewId)
+      return this.props.onInvalidViewId();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['onInvalidViewId' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4613375)